packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -492,20 +492,38 @@ class _CupertinoAlertDialogState extends State<CupertinoAlertDialog> {
   }
 }
 
-/// Rounded rectangle surface that looks like an iOS popup surface, e.g., alert dialog
-/// and action sheet.
+/// An iOS-style rounded rectangle surface useful for showing modal overlays,
+/// such as dialogs and action sheets.
 ///
-/// A [CupertinoPopupSurface] can be configured to paint or not paint a white
-/// color on top of its blurred area. Typical usage should paint white on top
-/// of the blur. However, the white paint can be disabled for the purpose of
-/// rendering divider gaps for a more complicated layout, e.g., [CupertinoAlertDialog].
-/// Additionally, the white paint can be disabled to render a blurred rounded
-/// rectangle without any color (similar to iOS's volume control popup).
+/// By default, this surface is composed of a transparent surface color painted
```

Comment:
Is "translucent" the more accurate word? IMO "transparent" means not being able to see at all.
```suggestion
/// By default, this surface is composed of a translucent surface color painted
```

Reply:
This is interesting and I did a little reading. Apparently translucent materials scatter light (e.g. like a blur convolution), whereas transparent materials do not (good demonstration: https://www.youtube.com/watch?v=jCnrne72W3U). Since the color is being painted on top of a blurred surface, but is not actually contributing to any pixel convolution, transparent would follow the strict definition. But to your point, "transparency" doesn't feel as right, so I'm on the wall on this one. Let me know what you think.

Reply:
I've also read some web pages that it seems people are not very consistent on it. Another problem is that, although the color alone is scatter-less, the overall effect of the pop up surface is with scatter. But I think using "transparent" here shouldn't cause much confusion anyway.